website/content/en/docs/ansible/development-tips.md

```diff
@@ -432,55 +461,54 @@ Declaring collections in the role meta allows you to invoke the
 ```
 
 ### Ansible Operator Conditions
-The Ansible Operator has a set of conditions which it will use as it performs
-its reconciliation procedure. There are only a few main conditions:
 
-* Running - the Ansible Operator is currently running the Ansible for
-  reconciliation.
+An Ansible Operator has a set of conditions that are used during reconciliation.
+There are mainly two condition types:
 
-* Successful - if the run has finished and there were no errors, the Ansible
-  Operator will be marked as Successful. It will then wait for the next
-  reconciliation action, either the reconcile period, dependent watches triggers
-  or the resource is updated.
+* `Running` - the Operator is currently running Ansible for reconciliation or
+  if a run has finished with no errors and it is waiting for the next
+  reconciliation action. The next action can be triggered by the reconcile
+  period, dependent watches or an update to the Custome Resource.
 
-* Failed - if there is any error during the reconciliation run, the Ansible
-  Operator will be marked as Failed with the error message from the error that
-  caused this condition. The error message is the raw output from the Ansible
-  run for reconciliation. If the failure is intermittent, often times the
-  situation can be resolved when the Operator reruns the reconciliation loop.
-
-Please look over the following sections for help debugging an Ansible Operator:
+* `Failure` - if there is any error during reconciliation, the Operator is
+  marked as `Failure` with the error message populated. The error message is
+  the raw output from the Ansible run. Usually, an intermittent failure is
+  automatically resolved when the Operator reruns the reconciliation.
 
+Please check the following sections for debugging an Ansible Operator:
 
 * [View the Ansible logs](../quickstart#view-the-ansible-logs)
 * [Additional Ansible debug](../quickstart#additional-ansible-debug)
 * [Testing Ansible Operators with Molecule](../testing-guide)
 
-## Extra vars sent to Ansible
-The extra vars that are sent to Ansible are managed by the operator. The `spec`
-section will pass along the key-value pairs as extra vars.  This is equivalent
-to how above extra vars are passed in to `ansible-playbook`. The operator also
-passes along additional variables under the `meta` field for the name of the CR
-and the namespace of the CR.
+## Extra Variables Sent to Ansible
+
+The Operator treats the content you provide in the `spec` section of the CR
+manifest as key-value pairs. It sends them as extra vars  to Ansible. This is
+equivalent to [passing extra vars to
+`ansible-playbook`](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#passing-variables-on-the-command-line).
+The operator also passes some additional key-value pairs using the `meta` key,
+e.g., the name and the namespace of the CR.
```

Comment:
```suggestion
The Operator treats the content you provide in the `spec` section of the CR
manifest as key-value pairs. It sends them as extra vars to Ansible and are managed by the operator. This is
equivalent to how above extra vars are passed into [passing extra vars to
`ansible-playbook`](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#passing-variables-on-the-command-line).
The operator also passes along additional variables under the `meta` field. These variable are key-value pairs, such as for example; the CR name and namespace. 
```

Reply:
> "and are managed by the operator."

What are managed by the operator? 

> The operator also passes along additional variables under the `meta` field. These variable are key-value pairs, such as for example; the CR name and namespace. 

I don't see any where key-value pairs are called "variables". It is confusing. "key-value pairs" contain both the variables and their values, aren't they?  "under the `meta` field" is also a weird expression ...

Reply:
Hi @tengqm, 

I am not native in English, however, shows that your modifications cange a little the understatement and has grammar issues.  I'd suggest revert and then do a PR with a close scope to address your motivations here. Ideally, we should not do a PR with so many different motivations.

Reply:
Em... alright. That makes sense.